tests/layer/test_gcnsupervisedgraphclassification.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018-2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from stellargraph.layer.graph_classification import *
+from stellargraph.mapper import GraphGenerator, FullBatchNodeGenerator
+import pytest
+from ..test_utils.graphs import example_graph_random
+
+
+class Test_GCNSupervisedGraphConvolution:
```

Comment:
Given the file already incorporates the type name, and, classes in pytest are purely an organisational thing (https://docs.pytest.org/en/latest/goodpractices.html#test-discovery "methods inside Test prefixed test classes **(without an __init__ method)**", i.e. the class can't do any special initialisation when being built), I think we could also just not have the class at all here or in the other file.

Since the `graphs` and `generator` values are small/cheap they can be promoted to the top-level and use `generator` instead of `self.generator`.  (If they were expensive to build, they should become fixtures so that they're only executed for tests that actually need them, not every time pytest searches for tests (e.g. #791); this applies with or without the class.)

(This is a small nit.)